client/src/hooks/useAutocomplete.js

```diff
@@ -115,6 +119,44 @@ const useAutocomplete = ({
     [immediateResponses, setImmediateResponseAutocomplete]
   );
 
+  const checkInvolvesChild = useCallback(
+    (desc) => {
+      if (involvesChildOptions) {
+        const involvesChild = autocompleteSingleOption(
+          desc,
+          involvesChildOptions
+        );
+        if (involvesChild) {
+          setInvolvesChildAutocomplete(involvesChild);
+        } else {
+          setInvolvesChildAutocomplete(
+            involvesChildOptions.find((o) => o.value === "no")
```

Comment:
Related to above comment - why do we iterate through the array here?

Reply:
The field options stored in Sanity would look something like: `[
    {
      "value": "no",
      "keywords": [],
      "name": "No"
    },
    {
      "value": "yes",
      "keywords": ["child", "children"],
      "name": "Yes"
    }
  ]`. The problem was the field option 'No' doesn't have any keywords to match against in the description, so I decided in case none of the other options' keywords were found in the description, it just defaults to 'No', which I know is an ugly way of doing it.
Lmk if you have any other suggestions

Reply:
I see. I don't think we would need to iterate through the options again - we should be fine passing in an object that mimics the option in the involvesChildOptions array. Something like `setInvolvesChildAutocomplete({ value: "no", name: "No" });` should probably work.

Even better, you can define a global object at the top of the file, something like `const NO_OPTION = { value: "no", name: "No" };` and then pass it into this function.

Reply:
Sure I made the changes.
The reason why I did it using .find was to just depend on the key and pull everything else from Sanity. But I guess this is just fine for in the case of `involvesChild` and `involvesGuest` fields.